packages/react/src/components/accordion/accordion-item.tsx

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Theme } from '../../themes';
+import { Button } from '../buttons/button';
+import { Icon } from '../icon/icon';
+import { Heading, Type, Tag } from '../heading/heading';
+
+interface AccordionItemProps {
+    title: string;
+    id?: string;
+    type?: Type | undefined;
+    tag?: Tag | undefined;
+    expanded?: boolean | undefined;
+    disabled?: boolean | undefined;
+    onToggle?: () => void;
+    onKeyDown?: ((event: React.KeyboardEvent<HTMLButtonElement>) => void) | undefined;
+    children: React.ReactNode;
+    buttonRef?: React.RefObject<HTMLButtonElement> | undefined;
+}
+
+const AccordionSection = styled.section<{ theme: Theme }>`
+    background: ${({ theme }) => theme.greys['colored-white']};
+    border-color: ${({ theme }) => theme.greys.grey};
+    border-radius: 0 0 var(--border-radius-2x) var(--border-radius-2x);
+    border-style: solid;
+    border-width: 0;
+    margin-bottom: var(--spacing-1x);
+    max-height: 0;
+    overflow: hidden;
+    transition: max-height 0.5s ease, border-width 0.5s ease;
+    &.expanded {
+        border-width: 1px;
+        border-top-width: 0;
+        max-height: 1000px;
+        overflow-y: scroll;
+        transform: translateZ(0);
+        transition: max-height 0.5s ease, border-width 0s ease;
+        will-change: max-height, border-width;
+    }
+`;
+
+const AccordionBody = styled.div<{ theme: Theme }>`
+    background: ${({ theme }) => theme.greys['colored-white']};
+    color: ${({ theme }) => theme.greys['neutral-90']};
+    font-size: 0.75rem;
+    font-weight: var(--font-normal);
+    letter-spacing: 0.015rem;
+    line-height: 1.7;
+    padding: var(--spacing-2x) var(--spacing-3x) var(--spacing-3x) var(--spacing-5x);
```

Comment:
There's a subtle difference in the `padding` due to the `margin` of the paragraph. 
![Screenshot 2023-09-14 at 11 02 33 AM](https://github.com/kronostechnologies/design-elements/assets/3722754/cce00088-d9d6-443f-8980-c4e0a148485b)

You might consider a solution like this:

```
> p:first-child {
        margin-top: 0;
    }
```
Feel free to explore other options as well. ;)

Reply:
I guess this will aways be an issue because users can add text and/or html elements.

Reply:
I believe it's more the responsibility of the user to eliminate these margins. As @carlafranca said, there is just an infinite possibility, you have many elements (H2, UL, TABLE, P, etc), and they can even be nested.
```
<div>
    <div>
        <p>
    <div>
</div>
```

For the tests, I think we should simply use plain text (or to the limit just div).

Reply:
fixed as per our conversation.